packages/flutter/lib/src/semantics/semantics.dart

```diff
@@ -48,6 +51,28 @@ typedef SetTextHandler = void Function(String text);
 /// Returned by [SemanticsConfiguration.getActionHandler].
 typedef SemanticsActionHandler = void Function(Object? args);
 
+/// The semantics merge rule generated from [SemanticsConfiguration.merger].
+class SemanticsMergeRule {
+  /// The child semantics configuration that are merging upward.
```

Comment:
Are the always merged up? What if they conflict with semantics information of the parent?